.github/CHANGELOG.md

```diff
@@ -2,6 +2,47 @@
 
 <h3>New features since last release</h3>
 
+* A transform `replace` has been added that allows you to define your own
```

Comment:
I worry about this general name - it would block anyone in future to ever use it for a transform, and there may be a good reason to use this short name elsewhere one day. 

Depending on the outcome of the discussion below, `custom_decomposition` could be cool?